readme.md

```diff
@@ -90,7 +90,7 @@ The following are **feature flags** that can be used with `--preset` flag (they
 
 ## 🔐 Repository secrets
 
-Some workflows generated by `react-native-ci-cli` require setting up repository secrets that can be then accessed by workflows as `${{ secrets.* }}`. Make sure to read logs printed by the script
+Some workflows generated by `setup-ci` require setting up repository secrets that can be then accessed by workflows as `${{ secrets.* }}`. Make sure to read logs printed by the script
```

Comment:
I wonder if we shouldn't always use npx setup-ci but let's discuss this on daily too

Reply:
we decided to use `npx setup-ci`